bridge/github/config.go

```diff
@@ -226,6 +237,16 @@ func promptTokenOptions(owner, project string) (string, error) {
 			return promptToken()
 		}
 
+		// Register restore state cleaner before prompting passwords inputs
+		termState, err := terminal.GetState(int(syscall.Stdin))
```

Comment:
with https://github.com/MichaelMure/git-bug/pull/210 done, you can now move those where `terminal.ReadPassword` is actually used.

Reply:
Done !